### PR TITLE
Some drags on scrolling "Let's talks section"

### PR DESCRIPTION
**Pull request by andriyParashchuk**
OS: iPhone SE
Browser: Chrome

On scrolling let's talk we have some drags.
Also will be great to fix wave animation based on design

Video: https://vimeo.com/266848542
Password: `ilikejt`

когда я скролю в хроме в разных направлениях
появляиться или прячиться топ бар - который менять размеры видимой части
и тут у нас запускаеться пересчитывание размера секций

### CODE REVIEW

**Comment by andriyParashchuk** [comment]
@pftg @obliviusm Pls check

File: _assets/javascripts/application.js [hunk 1]
```diff
@@ -4,6 +4,13 @@
 //= require form
 
 $(document).ready(function () {
+  var fullScreenHeight = function() {
+    $('.section').css('min-height', $(window).height());
+    $('.full-vh').css('height', $(window).height());
+  }
+  window.addEventListener('resize', fullScreenHeight);
```
Comment (pftg) [inline _assets/javascripts/application.js]: will this event be invoked when on mobile top bar will shows or hides?

Comment (andriyParashchuk) [reply _assets/javascripts/application.js]: this event will not work

**Comment by andriyParashchuk** [comment]
staging - http://jt-pr288.surge.sh/
@pftg  You can check

**Comment by pftg** [comment]
@obliviusm please verify code

**Comment by andriyParashchuk** [comment]
Update staging - http://jt-pr288.surge.sh/
Pls check

**Comment by obliviusm** [comment]
@andriyParashchuk 
1. there is red section at the bottom. on any phone
simulator screen shot - iphone 5s - 2018-05-08 at 17 02 46
2. Bug still happens. but only on iPhone 5s in Chrome browser. it works fine in Safari. same thing for production version. so this fix haven't changed anything

**Comment by andriyParashchuk** [comment]
Fix red section. 
Update staging - http://jt-pr288.surge.sh/
@obliviusm Psl check. If has scroling bug - make video pls

**Comment by pftg** [comment]
Has not been fixed yet

File: _assets/javascripts/application.js [hunk 2]
```diff
@@ -4,6 +4,15 @@
 //= require form
 
 $(document).ready(function () {
+  var fullScreenHeight = function() {
+    var windowHeight = $(window).height();
+    $('.js-min-full-vh').css('min-height', windowHeight);
+    $('.js-full-vh').css('height', windowHeight);
+  }
+  fullScreenHeight();
+  window.addEventListener('resize', fullScreenHeight);
```
Comment (pftg) [inline _assets/javascripts/application.js]: such events should have debouncer

**Comment by andriyParashchuk** [comment]
Deleted event on resize and add check for mobile device
staging - http://jt-pr288.surge.sh/
@pftg @obliviusm pls check

**Comment by obliviusm** [comment]
@andriyParashchuk has problem with an arrow
image

**Comment by pftg** [comment]
this arrow should be fixed in different pr

**Comment by andriyParashchuk** [comment]
Fixed
updated staging http://jt-pr288.surge.sh/
Pls check

**Comment by obliviusm** [comment]
@andriyParashchuk switch between vertical/horizontal screen doesn't work correctly on android

**Comment by andriyParashchuk** [comment]
Fixed
updated staging http://jt-pr288.surge.sh/
Pls check

**Comment by obliviusm** [comment]
@andriyParashchuk looks good. but it breaks height for tests.
can you fix it and try your staging url in ghost inspector (try one desktop and one mobile)? i suppose you can use your height function to re-write test code

**Comment by andriyParashchuk** [comment]
@obliviusm Fixed. pls check

**Pull request merged**